lib/ansible/plugins/inventory/netbox.py

```diff
@@ -134,7 +134,7 @@ class InventoryModule(BaseInventoryPlugin):
     NAME = 'netbox'
 
     def _fetch_information(self, url):
-        response = open_url(url, headers=self.headers, timeout=self.timeout)
+        response = open_url(url, headers=self.headers, timeout=self.timeout, validate_certs=False)
```

Comment:
Why is certificate validation being turned off?  Why not use the validate_certs parameter to the module?  I'd expect to see changes to parse() to honor the validate_certs argument and pass to open_url.  The examples should also be updated to include one example with this parameter.

Reply:
Done. Please have a look at the latest commit aa5022d06f0d3f764fd276f348cc4e3299d08d55